package.json

```diff
@@ -123,6 +124,7 @@
         "keytar": "^7.9.0"
     },
     "resolutions": {
-        "@types/node": "18.19.28"
+        "@types/node": "18.19.28",
+        "config-file-ts": "0.2.8-rc1"
```

Comment:
aaaaaaaaargh my kingdom for a config file format that lets you write COMMENTS

Reply:
For the benefit of anyone who gets here via PR spelunking: this is a workaround for https://github.com/electron-userland/electron-builder/issues/7775.